metricflow/test/integration/test_cases/itest_semi_additive_measure.yaml

```diff
@@ -0,0 +1,64 @@
+---
+integration_test:
+  name: semi_additive_measure_query_with_grouping
+  description: Tests selecting a measure with semi additive properties with a window_grouping
+  model: SIMPLE_MODEL
+  metrics: ["current_account_balance_by_user"]
+  group_bys: ["user"]
+  check_query: |
+    SELECT
+      a.user AS user
+      , SUM(a.current_account_balance_by_user) AS current_account_balance_by_user
+    FROM (
+      SELECT
+        ds
+        , user_id AS user
+        , account_balance AS current_account_balance_by_user
+      FROM (
+        SELECT * FROM {{ source_schema }}.fct_accounts
+      )
+    ) a
+    INNER JOIN (
+      SELECT
+        user_id AS user
+        , MAX(ds) AS ds
+      FROM (
+        SELECT * FROM {{ source_schema }}.fct_accounts
+      )
+      GROUP BY
+        user_id
+    ) b
+    ON
+      (a.ds = b.ds) AND (a.user = b.user)
```

Comment:
Guessing this might be from generated SQL - parenthesis are in there because we haven't put in operator precedence logic, but since people will be reading these, you can remove the parenthesis.